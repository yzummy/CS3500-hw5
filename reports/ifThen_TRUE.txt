
___Entering new scope...				      <
							      <
TOKEN: IF 		 LEXEME: if			      <
TOKEN: LPAREN 		 LEXEME: (			      <
TOKEN: TRUE 		 LEXEME: TRUE			      <
CONST -> TRUE						      <
FACTOR -> CONST						      <
TOKEN: RPAREN 		 LEXEME: )			      <
MULT_OP_LIST -> epsilon					      <
TERM -> FACTOR MULT_OP_LIST				      <
ADD_OP_LIST -> epsilon					      <
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST			      <
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC			      <
EXPR -> ARITHLOGIC_EXPR					      <
TOKEN: INTCONST 		 LEXEME: 42		      <
CONST -> INTCONST					      <
FACTOR -> CONST						      <
MULT_OP_LIST -> epsilon					      <
TERM -> FACTOR MULT_OP_LIST				      <
ADD_OP_LIST -> epsilon					      <
SIMPLE_ARITHLOGIC -> TERM ADD_OP_LIST			      <
ARITHLOGIC_EXPR -> SIMPLE_ARITHLOGIC			      <
EXPR -> ARITHLOGIC_EXPR					      <
IF_EXPR -> IF ( EXPR ) EXPR				      <
EXPR -> IF_EXPR						      <
START -> EXPR						      <
							      <
---- Completed parsing ----					---- Completed parsing ----

 Value of the expression is: 42				      |	Value of the expression is: 42
							      <
___Exiting scope...					      <
							      <
