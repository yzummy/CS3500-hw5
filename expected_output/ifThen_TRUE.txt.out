
---- Completed parsing ----

Value of the expression is: 42
